jujugui/static/gui/src/app/jujulib/bakery.js

```diff
@@ -71,8 +71,8 @@ var module = module;
       this._client = httpClient;
       this.storage = storage;
       this._onSuccess = params.onSuccess || (() => {});
-      this._visitPage = params.visitPage || (visitURL => {
-        window.open(visitURL, 'Login');
+      this._visitPage = params.visitPage || (error => {
```

Comment:
Please update the docstring for the constructor, and describe the fields available to the object passed to visitPage.